recipes/osqp/all/conanfile.py

```diff
@@ -18,12 +18,29 @@ class OsqpConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "printing": [True, False],
+        "profiling": [True, False],
+        "ctrlc": [True, False],
+        "dfloat": [True, False],
+        "dlong": [True, False],
+        "coverage": [True, False],
```

Comment:
`coverage` does not really make much sense as a package option. It only has an effect when unit tests are being run.

Reply:
Yes, as a library user I don't really care about coverage so happy to skip (done in commits I just pushed), but as the `UNITTEST` option was handled by the recipe possibly the `COVERAGE` could be too.